keel-core-test/src/main/kotlin/com/netflix/spinnaker/keel/persistence/ArtifactRepositoryTests.kt

```diff
@@ -649,7 +650,10 @@ abstract class ArtifactRepositoryTests<T : ArtifactRepository> : JUnit5Minutests
     }
 
     context("artifact creation timestamp exists") {
-      val createdAt = Instant.now()
+      // We truncate this since we're using a serialization to java that reduces the level of precision
+      // and later comparisons break otherwise.  This is needed to work with generated columns in
+      // certain databases.  See the PrecisionSqlSerializer class for more info
+      val createdAt = Instant.now().truncatedTo(ChronoUnit.MILLIS)
```

Comment:
Now that we're using 6 digits, can we change it to ChronoUnit.MICROS?

Reply:
Key question I see is do we want 6 or 3?  Database for these columns is 3.  BUT mysql can support 6.  I'd lean towards leaving it 6, and just accept that those specific generated columns only do 3 (IF I'm reading the spec on the `datetime(3)` correctly

Reply:
And I'd be good with MICROS :)